go/examples/aws-lambda-source/test-receiver/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+type MyEvent struct {
+	Name string `json:name`
+	Age  int    `json:age`
+}
+
+type MyResponse struct {
+	Message string `json:"Answer:"`
```

Comment:
variable name should be the same as json key

Reply:
this is copy&paste from AWS sample code, but i can change it...

Reply:
removed the test code